css/styles.css

```diff
@@ -1,5 +1,9 @@
-@media (min-width: 1200px) {
+/* @media (min-width: 1200px) {
```

Comment:
No need to override the widths of containers. Using width !important on it could interfere with bootstrap's original implementation. Try to avoid using !important in general.